### ISSUE TITLE
[Suggestion] Use some kind of "split ID" rather than split name when comparing during SRL races

### ISSUE THREAD

**Issue by jeffreyhobson** [LiveSplit/LiveSplit#115]
I absolutely LOVE the SRL race features of LiveSplit, but don't get to compare against my opponents on as many splits as I would prefer due simply to having different split names (e.g. I use roman numerals for multi-part bosses but many opponents don't). Having a place to enter split IDs (similar to SRL game IDs) would be a very convenient solution to this problem, even if it weren't exactly the most intuitive solution for less-technical users. This way game communities could keep a set list of split IDs for anybody to implement so they could race against fellow runners!

**Comment by CryZe** [comment]
I don't think that would work out that well, as most people wouldn't care about filling that out properly. We'll most likely make it possible to define a list of alternate split names and that gets shared among all racers. So if different people have a small list, it gets combined into a large list. Also, if only a single person filled it out, it still works for everyone.

**Comment by jeffreyhobson** [comment]
That would work fine as well. Honestly it's basically what I was envisioning anyways. My idea of split IDs was that it would default to using split names if no ID is set anyways (maybe "race split name" or "segment race ID" would be a better name). So for example, my first few splits for Kingdom Hearts would look like this in my envisioning (and probably pretty similar in the way you're describing):

In the Edit Splits... dialog:
Icon | Race ID | Segment Name | Split Time ... etc.
   x   | dksd1    | Darkside I | ...
   x   |              | Day II | <-- A split many people omit, thus I let it default to split name
   x   | dksd2    | Darkside II | ...
   x   | leon       | Leon | ...
   x   | grdarmr | Guard Armor | <-- A split often spelled differently per region, thus race ID helps
etc.

All in all, we're basically talking two sides of the same coin here if I'm understanding you correctly, I just think shorter IDs might be a bit nicer than full alternate split names.

**Comment by jeffreyhobson** [comment]
Just thought of another enhancement IDs or alternate names could provide: Use them when importing a new comparison (defaulting to split names if IDs don't match or aren't provided). Basically same reasoning as for races, but figured I'd mention it again here.

**Comment by CryZe** [comment]
Yeah, it uses the same code, so importing a comparison would use it too.

**Comment by glacials** [comment]
Some discussion about this over at glacials/splits-io#190 -- @CryZe, to address concerns about people not wanting to fill out split IDs, splits.io could autogenerate them for uploaded routes (see glacials/splits-io#67), so anyone who downloads that route uses the same split IDs as everyone else who downloaded it.

This'd be a bit of a ways off though, as Routes isn't even a feature yet, just tossing the idea around.
